src/main/java/com/zoo/species/Eagle.java

```diff
@@ -1,4 +1,20 @@
 package com.zoo.species;
 
-public class Eagle {
-}
+import com.zoo.animals.Bird;
+
+public class Eagle extends Bird {
+
+    public Eagle(int energyLevel, String name, int age, int weight) {
+        super(energyLevel, name, age, weight);
+    }
+
+    public void makeSound() {
+        System.out.println("Кріі-кріі!");
+    }
+
+    public void fly() {
+        int currentEnergy = getEnergyLevel();
```

Comment:
btw maybe this could've been done by introducing a protected method for Animal like `protected void decreaseEnergyLevelBy(int energyPoints)`